### DISCUSSION TITLE
Disable side menu collapse button

### DISCUSSION THREAD

**Discussion by almenscorner** [kikipoulet/SukiUI#209] [Q&A] [answered]
Hi,

Is it possible to disable the collapse button on the side menu to always keep it open?

**Comment by AuroraZiling** [comment, marked as answer]
the button is called PART_SidebarToggleButton in SukiSideMenu

You can try to set its property IsVisible=false from styles in your app to hide it

**Reply by almenscorner** [reply]
Thanks, I was able to hide it with
```
    <controls:SukiWindow.Styles>
        <Style Selector="Button#PART_SidebarToggleButton">
            <Setter Property="IsVisible" Value="False"/>
        </Style>
    </controls:SukiWindow.Styles>
```

**Reply by AuroraZiling** [reply]
next beta will support to set SukiSideMenu IsToggleButtonVisible=false :)
